src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -36,9 +36,19 @@ const getBookResponse = async(
   osWebLoader: AppServices['osWebLoader'],
   archiveLoader: AppServices['archiveLoader'],
   loader: ReturnType<AppServices['archiveLoader']['book']>,
+  dispatch: Dispatch,
+  bookId: string,
   bookSlug?: string
 ): Promise<[Book, ReturnType<AppServices['archiveLoader']['book']>]>  => {
   const osWebBook = bookSlug ? await osWebLoader.getBookFromSlug(bookSlug) : undefined;
+
+  const bookConfig = getBookVersionFromUUIDSync(bookId);
+  // if (bookConfig?.retired && !UNLIMITED_CONTENT) {
+  if (bookConfig?.retired) {
+    dispatch(receivePageNotFoundId(null));
+    return Promise.reject();
+  }
```

Comment:
@TomWoodward however I try to bail here I end up with a "Book not loaded" error from the intlHook